### ISSUE TITLE
Move man page to /usr/share/man

### ISSUE THREAD

**Issue by vestingz** [cfv-project/cfv#47]
cfv seems to ship manpages to `/usr/man/man1`, following a really old standard from [1995](http://www.ibiblio.org/pub/Linux/docs/fhs/old/fsstnd-1.2.txt), which has already been superseded in [1997](http://www.ibiblio.org/pub/Linux/docs/fhs/fhs-2.0.tar.gz). The [current standard](https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html#usrsharemanManualPages) suggests to keep man pages in `/usr/share/man`. cfv should follow this standard.

**Comment by lxp** [comment]
Could this be an Arch Linux specific issue?

When I run `pip install cfv` on a fresh Ubuntu 22.04 the man page is installed to `/usr/local/share/man/man1/cfv.1`, which seems fine to me.

**Comment by vestingz** [comment]
I am not quite familiar with the installation process in python. Could it be possible that Ubuntu generally symlinks `/usr/man/man1` to `/usr/local/share/man/man1`?

**Comment by lxp** [comment]
I don't think so, `/usr/man` does not exist at all on Ubuntu 22.04.
This is what I tried:

```sh
$ docker run --rm -it ubuntu:22.04
root@09191a7797be:/# apt update
[...]
root@09191a7797be:/# apt install python3-pip
[...]
root@09191a7797be:/# pip install cfv
Collecting cfv
  Downloading cfv-3.0.0-py3-none-any.whl (49 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 49.1/49.1 KB 1.3 MB/s eta 0:00:00
Installing collected packages: cfv
Successfully installed cfv-3.0.0
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
root@09191a7797be:/# ls -lah /usr/man 
ls: cannot access '/usr/man': No such file or directory
root@09191a7797be:/# ls -lah /usr/local/share/man/man1/cfv.1 
-rw-r--r-- 1 root root 13K Nov 10 22:57 /usr/local/share/man/man1/cfv.1
root@09191a7797be:/# namei -l /usr/local/share/man/man1/cfv.1 
f: /usr/local/share/man/man1/cfv.1
drwxr-xr-x root root /
drwxr-xr-x root root usr
drwxr-xr-x root root local
drwxr-xr-x root root share
drwxr-xr-x root root man
drwxr-xr-x root root man1
-rw-r--r-- root root cfv.1
```

I think the final install location is chosen by pip in this case. I am not sure how your installation works and what system/versions you use. Maybe you can provide a similar reproducer.

**Comment by vestingz** [comment]
The final installation location is actually determined by the [`data_files`](https://github.com/cfv-project/cfv/blob/636a20399108d2a2332fd6d0430ec29335c4ad6a/setup.py#L63) directive in `setup.py`. So I guess it would suffice to change that from `man/man1` to `share/man`? 

`/usr` should be omitted according to the [documentation](https://docs.python.org/3/distutils/setupscript.html#installing-additional-files).

**Comment by lxp** [comment]
I analysed it again, you where right with your guess that there is a link in Ubuntu. Thanks for your patience :)
At first I just looked for `/usr/man`, but Ubuntu defaults to installing pip packages to `/usr/local`, so the link was located there:
```
lrwxrwxrwx 1 root root 9 Nov  1 21:15 /usr/local/man -> share/man
```
Just for the fun, ArchLinux has this link in the opposite direction:
```
lrwxrwxrwx 1 root root 6 Oct 18 21:01 /usr/local/share/man -> ../man
```
However, ArchLinux defaults to installing pip packages to `/usr`.
`/usr/man` does not exist on both distros.

I think you are right that we should just stick to the newer standard of installing to `share/man/man1` instead of `man/man1`.
